src/test/java/com/frameworkium/integration/heroku/theinternet/pages/web/SortableDataTablesPage.java

```diff
@@ -27,6 +27,10 @@
     @FindBy(css = "table#table2")
     private Table table2;
 
+    @Visible(atLeast = 2)
```

Comment:
might be better to add the `atLeast` param to the list in `CheckboxesPage` rather than create a new field which isn't used here.